linkcheck/director/aggregator.py

```diff
@@ -17,10 +17,7 @@
 Aggregate needed object instances for checker threads.
 """
 import threading
-try:  # Python 3
-    import _thread
-except ImportError:
-    import thread as _thread
+import _thread
```

Comment:
This is used only for `_thread.get_ident()`.

I'm pretty sure https://docs.python.org/3/library/threading.html#threading.get_ident is the public alias for it.

> New in version 3.3.

yeah, we can use it.

Reply:
Also done.